mgmt/rest/v2/error.go

```diff
@@ -0,0 +1,76 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"fmt"
+
+	"errors"
+
+	"github.com/intelsdi-x/snap/core/serror"
+)
+
+const (
+	ErrPluginAlreadyLoaded     = "plugin is already loaded"
```

Comment:
Let's take this opportunity to make the errors consistent in terms of language, punctuation and capitalization.